frontend/src/actions/filterActions.js

```diff
@@ -91,6 +92,9 @@ export const setDateFilter = (date, key, navigation, currType) => {
     ilab: applyIlabDateFilter,
   };
   dispatch(actions[currType](date, key, navigation));
+  if (currType === "ilab") {
+    dispatch(fetchILabJobs(true));
+  }
```

Comment:
Isn't this handled somewhere else now?? I don't think there's any reason for your fix to be changing this code...